client/components/jetpack/jetpack-welcome-page/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Card } from '@automattic/components';
+import classnames from 'classnames';
+import JetpackLogo from 'calypso/components/jetpack-logo';
+import Main from 'calypso/components/main';
+
+import './style.scss';
+
+export type JetpackWelcomePageProps = {
```

Comment:
We probably could not export this type as it's used only in the file

Reply:
👍